firefox-ios/Client/Frontend/Browser/Tabs/Views/TabDisplayView.swift

```diff
@@ -78,14 +77,12 @@ class TabDisplayView: UIView,
 
     public init(panelType: TabTrayPanelType,
                 state: TabsPanelState,
-                windowUUID: WindowUUID,
-                animationQueue: TabTrayAnimationQueue = TabTrayAnimationQueueImplementation()) {
```

Comment:
At a glance it looks like this is the only place that `TabTrayAnimationQueueImplementation` was used? NAB for this PR but we could probably also remove the related `TabTrayAnimationQueue.swift` file.